src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf

```diff
@@ -301,11 +301,35 @@
             <trans-unit id="78">
                 <source>An empty file is not allowed.</source>
                 <target>No està permès un fixter buit.</target>
+            </trans-unit>
+              <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>No s'ha pogut resoldre l'amfitrió.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Aquest valor no coincideix amb l'esperat {{ charset }} joc de caràcters.</target>
+            </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>Aquest no és un codi d'identificació bancari (BIC) vàlid.</target>
+            </trans-unit>
+            <trans-unit id="82">
+                <source>Error</source>
+                <target>Error</target>
             </trans-unit>
             <trans-unit id="83">
                 <source>This is not a valid UUID.</source>
                 <target>Aquest valor no és un UUID vàlid.</target>
             </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Aquest valor ha de ser múltiple de {{ compared_value }}</target>
```

Comment:
There is a missing dot at the end of this translation. Thanks.

Reply:
Fiexed